algorithm/include/gnuradio-4.0/algorithm/fourier/fft_types.hpp

```diff
@@ -1,28 +1,12 @@
 #ifndef GNURADIO_ALGORITHM_FFT_TYPES_HPP
 #define GNURADIO_ALGORITHM_FFT_TYPES_HPP
 
-#include <type_traits>
 #include <complex>
+#include <type_traits>
```

Comment:
Is this include (nearly) empty? If yes, please remove...

Reply:
Basically there is only one concept definition inside for complex numbers:
```cpp
concept ComplexType = std::is_same_v<T, std::complex<float>> || std::is_same_v<T, std::complex<double>>;
```

But it is widely in use in FFT classes. 
Where can I move it? What would you propose?